app/javascript/src/components/Dashboard/Notes/Note/index.jsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
 
-import NoteContainer from "./Note/NoteContainer";
-import NoteContent from "./Note/NoteContent";
-import NoteFooter from "./Note/NoteFooter";
-import NoteHeader from "./Note/NoteHeader";
+import NoteContainer from "./NoteContainer";
+import NoteContent from "./NoteContent";
+import NoteFooter from "./NoteFooter";
+import NoteHeader from "./NoteHeader";
```

Comment:
Just name the components as `Container`, `Content` and avoid the `Note` prefix since these components are already within the `Note` namespace.

In any case where there is conflict, then import the component with the namespace, example:
```js
import NoteContainer from "./Note/Container";
```

Reply:
@yedhink 
I have made the mentioned changes. Please have a look at the last commit.